__docs__/wonder-blocks-form/text-area.stories.tsx

```diff
@@ -0,0 +1,444 @@
+            console.log("Default Wrap:");
+            console.log(formData.get("default-wrap"));
+            /* eslint-enable no-console */
+        };
+        return (
+            <div>
+                <p>
+                    Once submitted, check the console to observe the wrapping
+                    behaviour in the form data
+                </p>
+                <form
+                    method="POST"
+                    action="/submit"
+                    id="wrapForm"
+                    onSubmit={handleSubmit}
+                    style={{width: "300px"}}
+                >
+                    <label htmlFor="soft-wrap">Wrap: soft</label>
+                    <TextArea
+                        {...args}
+                        wrap="soft"
+                        id="soft-wrap"
+                        name="soft-wrap"
+                    />
+                    <br />
+                    <label htmlFor="hard-wrap">Wrap: hard</label>
+                    <TextArea
+                        {...args}
+                        wrap="hard"
+                        id="hard-wrap"
+                        name="hard-wrap"
+                    />
+                    <br />
+                    <label htmlFor="off-wrap">Wrap: off</label>
+                    <TextArea
+                        {...args}
+                        wrap="off"
+                        id="off-wrap"
+                        name="off-wrap"
+                    />
+                    <br />
+                    <label htmlFor="default-wrap">Wrap: default (soft)</label>
+                    <TextArea {...args} id="default-wrap" name="default-wrap" />
+                    <br />
+                    <Button type="submit">Submit</Button>
+                </form>
+            </div>
+        );
+    },
+};
+
+/**
+ * The `minlength` and `maxlength` textarea attributes can be set using the
+ * `minLength` and `maxLength` props.
+ *
+ * Note: At this time, character length requirements are not displayed as part of
+ * the Text Area component. These props are only setting the underlying HTML
+ * attributes ([minlength](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/minlength)
+ * and [maxlength](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/maxlength)).
+ */
+export const MinMaxLength: StoryComponentType = {
+    args: {
+        minLength: 2,
+        maxLength: 4,
+        value: "Text",
+    },
+    render: ControlledTextArea,
+};
+
+/**
+ * The behaviour of the built-in resize control can be configured using the
+ * `resizeType` prop. Here are some tips:
+ * - The initial size of the TextArea can be configured using the `rows` prop.
+ * This size should be large enough for the expected user input.
+ * - Avoid having too small of a TextArea and having `resizeType=none`. This
+ * makes it difficult for users to scroll through their input.
+ */
+export const ResizeType: StoryComponentType = {
```

Comment:
**suggestion:** I think this snapshot can be skipped from Chromatic.

Reply:
I left this one to be tested by Chromatic to verify the resizing control types in case styling impacts it later on!